components/infobox/commons/infobox_team.lua

```diff
@@ -189,7 +189,7 @@ end
 
 function Team:_setLpdbData(args, links)
 	local name = args.romanized_name or self.name
-	local earnings = _earnings.total
+	local earnings = Variables.varDefault('earnings')
```

Comment:
1) Isn't the value available is without Variables?

Reply:
would have to add a
`local _total_earnings` at the top for it, but the var is needed for smw anyways (until we kick that shit)

Reply:
I think there's an advantage to have the same code logic in the Earnings between the Infobox Player and Infobox Team, and Players used the a class variable?